src/pages/TasksManagementPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+// import { Helmet, formatMessage } from '@openimis/fe-core';
+import { useSelector } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
+import { RIGHT_TASKS_MANAGEMENT_SEARCH } from '../constants';
+import BenefitPlanTasksPage from './BenefitPlanTasksPage';
+
+const styles = (theme) => ({
+  page: theme.page,
+  paper: theme.paper.paper,
+  title: theme.paper.title,
+});
+
+function TasksManagementPage() {
+  const rights = useSelector((store) => store.core?.user?.i_user?.rights ?? []);
+
+  return (
+    rights.includes(RIGHT_TASKS_MANAGEMENT_SEARCH) && (
+      <>
+        <BenefitPlanTasksPage rights={rights} />
+        {/* REST OF PAGES */}
+      </>
+    )
+  );
```

Comment:
As mentioned for the BenefitPlanTasks this should be handled by contribution point

Reply:
Moving all benefit plan logic to the other modules will be handled in scope of the [CM-139](https://openimis.atlassian.net/browse/CM-139).